Jenkinsfile-deploy

```diff
@@ -0,0 +1,8 @@
+library("tdr-jenkinslib")
```

Comment:
Indentation should be 2 spaces. Looks like 4 spaces.

Reply:
I think it was 6 oddly. I've changed it to 2 spaces in this file and the other one.